src/FormHandler/HandlerConfigInterface.php

```diff
@@ -52,6 +52,18 @@ public function onSuccess(callable $callback);
     public function onFailure(callable $callback);
 
     /**
+     * Register the FormSubmitProcessor
+     *
+     * This will overwrite the default FormSubmitProcessor.
+     *
+     * signature:
+     * function (FormInterface $form, Request $request): void;
+     *
+     * @param callable $callback
+     */
+    public function setFormSubmitProcessor(callable $callback);
```

Comment:
Don't forget to add a BC notice about this

Reply:
done